public/static/locales/en/intro.json

```diff
@@ -0,0 +1,24 @@
+{
+    "analysesTitle": "Analyses",
+    "appsTitle": "Apps",
+    "dashboardTitle": "Dashboard",
+    "dataTitle": "Data",
+    "notificationTitle": "Notifications",
+    "preferencesTitle": "Preferences",
+    "bagTitle": "Bags",
+    "searchTitle": "Search",
+    "supportTitle": "Support",
+    "menuTitle": "Menu",
+    "menu": "<p>Use this menu to access your Analyses, Apps and Data or to return to Dashboard from anywhere within the DE. Click to open the menu.</p>",
+    "analyses": "<p>Use the Analyses View to find information about the analyses (jobs) you have launched.</p><p>In the Analyses View, you can:<ul><li>Check the status of an analysis.</li><li>Access the results of your analyses.</li><li>View the parameters you set for each analysis.</li><li>Relaunch an analysis using all of the options (parameters) you previously selected.</li></ul></p>",
+    "apps": "<p>Use the Apps View to view and find apps to analyze your data.</p><p>In the Apps View, you can: <ul><li>Browse for all available apps in CyVerse.</li><li>Customize an existing app.</li><li>Create your own App.</li></ul></p>",
+    "data": "<p>Use the Data view to manage your data in our cloud-based, central repository called the CyVerse Data Store.<ul><li>You can transfer data in/out of CyVerse.</li><li>Access data you own, data shared with you, and public data.</li><li>Share your data with other CyVerse or external users.</li></ul></p>",
+    "notifications": "<p>Automated notifications let you track progress of actions, such as: <ul><li>The status of your analyses, data transfers, and any requests you have made.</li><li>Sharing of files and folders with you or by you.</li></ul></p>",
+    "preferences": "<p>Use the Preferences to customize settings for your DE experience.<ul><li>Customize notification preferences.</li><li>Change settings and modify keyboard shortcuts.</li></ul></p>",
+    "bag": "<p>Use a bag to perform bulk operations. <ul><li>Share Analyses, Apps, and Data from the bag.</li><li>Download multiple data sets from the bag.</li></ul></p>",
+    "dashboard": "<p>Use Dashboard to find information about: <ul><li>Recently launched analyses.</li><li>Recently used Apps.</li><li>New Apps published in the DE.</li><li>CyVerse New and Events.</li></ul></p>",
+    "search": "<p>Use Global Search to: <ul><li>Type 3 or more characters to quickly search and find Analyses, Apps and Data.</li><li>Filter search results by type.</li><li>View detailed results page by hitting the `Enter` key followed by the search term or selecting the `View All` from the results.</li></ul></p>",
```

Comment:
To me, this sounds like you're supposed to hit Enter and then type out a search term.  Maybe `... by hitting the Enter key after the search term`? 
Also for `... or selecting the View All from ...` - I would either drop `the` from this sentence or keep `the` and add what `View All` is - such as `View All link` or `View All option`